README.md

```diff
@@ -31,13 +31,13 @@ Optuna can dynamically construct the search spaces for the hyperparameters.
 
 Optuna has modern functionalities as follows:
 
-- [Lightweight, versatile, and platform agnostic architecture](https://optuna.readthedocs.io/en/stable/tutorial/001_first.html)
+- [Lightweight, versatile, and platform agnostic architecture](https://optuna.readthedocs.io/en/stable/tutorial/10_key_features/001_first.html)
   - Handle a wide variety of tasks with a simple installation that has few requirements.
-- [Pythonic search spaces](https://optuna.readthedocs.io/en/stable/tutorial/002_configurations.html)
+- [Pythonic search spaces](https://optuna.readthedocs.io/en/stable/tutorial/10_key_features/002_configurations.html)
   - Define search spaces using familiar Python syntax including conditionals and loops.
-- [Efficient optimization algorithms](https://optuna.readthedocs.io/en/stable/tutorial/007_pruning.html)
+- [Efficient optimization algorithms](https://optuna.readthedocs.io/en/stable/tutorial/10_key_features/003_efficient_optimization_algorithms.html)
   - Adopt state-of-the-art algorithms for sampling hyper parameters and efficiently pruning unpromising trials.
-- [Easy parallelization](https://optuna.readthedocs.io/en/stable/tutorial/004_distributed.html)
+- [Easy parallelization](https://optuna.readthedocs.io/en/stable/tutorial/10_key_features/004_distributed.html)
   - Scale studies to tens or hundreds or workers with little or no changes to the code.
 - [Quick visualization](https://optuna.readthedocs.io/en/stable/reference/visualization/index.html)
   - Inspect optimization histories from a variety of plotting functions.
```

Comment:
We have a tutorial page for the visualization feature now, and I think we can replace this like with it.
Should we work on it in a follow-up PR?

https://optuna.readthedocs.io/en/stable/tutorial/10_key_features/005_visualization.html

Reply:
You're right. It's also already updated at the [key features in the docs](https://optuna.readthedocs.io/en/stable/). I just fixed the URL.